x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/watcher/history/WatchRecord.java

```diff
@@ -179,6 +183,9 @@ public final XContentBuilder toXContent(XContentBuilder builder, Params params)
         if (executionResult != null) {
             builder.field(EXECUTION_RESULT.getPreferredName(), executionResult, params);
         }
+        if (user != null) {
```

Comment:
can you maybe move this up to where the node id gets written out? This way, one can see both at once when reading a watch history, otherwise one is at the top and the other at the bottom.

Reply:
Missed, will do this in another PR.

Reply:
oh nm, i can do it on this one! I thought i had merged already woop woop

Reply:
@spinscale can you eyeball 0dc862f just so im sure i did what u asked?